frontera/contrib/messagebus/zeromq/broker.py

```diff
@@ -107,23 +108,23 @@ def handle_db_out_recv(self, msg):
 
     def handle_db_in_recv(self, msg):
         self.stats['db_in_recvd'] += 1
-        if msg[0][0] in ['\x01', '\x00']:
+        if to_native_str(msg[0])[0] in ['\x01', '\x00']:
```

Comment:
msg[0] is bytes always. The comparison works in PY2, but in PY3 the characters of the bytestring are interpreted as integers so `1 in ['\x01', '\x00']` fails. Operations such as these require unicode in PY3.